packages/oui-switch/src/switch.component.js

```diff
@@ -0,0 +1,23 @@
+import controller from './switch.controller'
+
+export default {
+  template: `<label class="oui-switch">
```

Comment:
What are the usecases of a switch compared to a checkbox ?

Can `onChange` event be async ?

Reply:
No, but it could be, just specify in which use case here please (even if I can guess it).

Reply:
What are the usecases of a switch compared to a checkbox ? 
We often use switches if the action is immediate (1 click = 1 post / put)

Reply:
Yes @Mboulianne , it's the answer i would like to read :p

Switches -> immediate so async event won't be possible i think or we need to manage loader in switch

Reply:
yeah but why not deal with this specific issue when we encounter it?

Reply:
Or loader instead of switch...
I also think about the case of error: the use turns on the switch, the post request returns an error so that the switch should be turned back off.
The question is: is this logic must be supported by the component itself or by the controller (outside of the component)?